tasks/aws-elasticache.yml

```diff
@@ -0,0 +1,121 @@
+---
+- name: Ensure dev packages are installed.
+  apt:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - zlib1g-dev
+    - pkg-config
+    - libevent-dev
+    - gcc
+    - g++
+    - make
+    - autoconf
+    - libsasl2-dev
+    - git
+
+- name: Install php-igbinary if PHP>5.
+  apt:
+    name: "php-igbinary"
+    state: present
+  when: php_version_to_install != 5
+
+- name: Install php-igbinary if PHP5.
+  apt:
+    name: "php5-igbinary"
+    state: present
+  when: php_version_to_install == 5
+
+- name: Check that /tmp/aws-libmemcached exist
+  stat:
+    path: /tmp/aws-libmemcached
+  register: _aws_libmemcached
+
+- name: Clone the aws-libmemcached repo.
+  git:
+    repo: https://github.com/awslabs/aws-elasticache-cluster-client-libmemcached
+    dest: /tmp/aws-libmemcached
```

Comment:
Thinking in my case:
I have an ASG. I run php role during the build of the ami to setup base packages.
Then I also run the role at boot to have php configuration based on the memory of my instance.
Because of the /tmp path I will have each time the git clone right ?
Maybe we could put it in something like /usr/src/ ?

Reply:
You're right, I forgot we were running the php role at boot again (but make sense for optimization with the cpu)